pkg/model/awsmodel/network.go

```diff
@@ -468,7 +468,7 @@ func (b *NetworkModelBuilder) Build(c *fi.CloudupModelBuilderContext) error {
 					Tags:      nil, // We don't need to add tags here
 				}
 
-				c.AddTask(in)
+				c.EnsureTask(in)
```

Comment:
Please check the error returned by `EnsureTask`.

Reply:
There is none anymore, which is great. 😄 
https://github.com/kubernetes/kops/pull/14927